### PR TITLE
Make console

### PR DESCRIPTION
**Pull request by SamCitron**
Fixes CIMPLE-408 linked [here](https://standardhealthrecord.atlassian.net/secure/RapidBoard.jspa?rapidView=9&projectKey=INT&modal=detail&selectedIssue=CIMPL-408&assignee=5ece9e7747d31e0c2a0e7681). Displays SUSHI logger in console on screen. However, displays all text at once, and is not yet color coded. Console clears with every run.

### CODE REVIEW

File: src/components/ConsoleComponent.js [hunk 1]
```diff
@@ -7,16 +7,27 @@ const useStyles = makeStyles((theme) => ({
     padding: theme.spacing(2),
     color: theme.palette.common.white,
     background: theme.palette.common.black,
-    height: '100%',
+    height: '200%',
```
Comment (jafeltra) [inline src/components/ConsoleComponent.js:10]: Why did you need to change this height? Just curious!

File: src/components/TopBar.js [hunk 2]
```diff
@@ -5,17 +5,20 @@ import { AppBar, Toolbar, Button, Typography } from '@material-ui/core';
...
-    margin: theme.spacing(2)
+    margin: theme.spacing(1),
+    color: 'white',
+    background: 'linear-gradient(45deg, #30638E 30%, #285a85 90%)',
+    boxShadow: '0 3px 5px 2px #285a85'
   },
   title: {
     flexGrow: 1,
-    edge: 'start',
-    fontFamily: 'Consolas'
```
Comment (jafeltra) [inline src/components/TopBar.js:18]: Did you remove the `fontFamily` here as part of matching the top bar with FSH School? I'm just curious, especially since we were talking about the different fonts earlier.

File: src/components/ConsoleComponent.js (hunk 1 above)
Comment (SamCitron) [reply src/components/ConsoleComponent.js:10]: I just felt it looked better, without the increased height the console can be a bit narrow and you can't see entirely the box. However I can totally change it back if we would prefer it fit on one page

File: src/components/TopBar.js (hunk 2 above)
Comment (SamCitron) [reply src/components/TopBar.js:18]: Yes, I was trying to get it to most closely resemble the font used in the fsh-school or fsh-online.

Comment (jafeltra) [reply src/components/TopBar.js:18]: Ah cool! Makes sense to me!

File: src/components/ConsoleComponent.js (hunk 1 above)
Comment (jafeltra) [reply src/components/ConsoleComponent.js:10]: I think maybe ideally everything would fit on one page without needing to scroll, but it needed to scroll before anyway, and I think you're right, this is better for now at least because you can see more of the console!
